core/flyout_vertical.ts

```diff
@@ -274,7 +274,7 @@ export class VerticalFlyout extends Flyout {
    * @param currentDragDeltaXY How far the pointer has moved from the position
    *     at mouse down, in pixel units.
    * @returns True if the drag is toward the workspace.
-   * @internal
+   *
```

Comment:
Whoops, this blank line in the comment is a lint error, could you remove it?

Reply:
Hello @johnnesky  my bad. yes i  just removed and pushed my changes